pulumi/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,10 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$bindir   = Join-Path -Path $toolsDir -ChildPath "bin"
+
+Uninstall-ChocolateyPath $bindir 'Machine'
+
+if (Test-Path $toolsDir) {
+    Remove-Item $toolsDir -Recurse -Force
```

Comment:
Remove the $toolsDir\Pulumi folder, not the toolsDir self. This also contains your uninstall script.